pabi_account_financial_report_webkit_xls/wizard/general_ledger_wizard.py

```diff
@@ -8,8 +8,27 @@ class general_ledger_webkit_wizard(orm.TransientModel):
     _inherit = 'general.ledger.webkit'
 
     def xls_export(self, cr, uid, ids, context=None):
+        self._update_date_reconciled(cr, uid, ids, context=context)
```

Comment:
In report, there shouldn't be any updates.

Reply:
update "date_reconciled" after Generate Fiscal Year Opening Entries 

change to PR: https://github.com/pabi2/pb2_addons/pull/1802